.github/workflows/weekly.yml

```diff
@@ -0,0 +1,61 @@
+name: Weekly Test Workflow for Released Branches
+on:
+  schedule:
+    - cron: '0 6 * * 0'
```

Comment:
Is this going to consume 100% of our test workers?  Is there a way to make these run sequentially instead of in paralle?